src/aliceVision/sfm/pipeline/panorama/ReconstructionEngine_panorama.cpp

```diff
@@ -438,9 +440,95 @@ bool ReconstructionEngine_panorama::Adjust()
       return false;
   }
 
+  //Assuming only a small number of views have outliers due to motion
```

Comment:
It would be more clear to me to have the major action here, like the fact that we restore the initial poses (here hidden in the "cleanWithPriors").
And explain the global idea, if I understand correctly: we are now considering the intrinsics resolved. With this new constraint, we restart from the inital poses with more strict matches to estimate the relative poses to avoid outlier (like the usual problem of moving clouds).